src/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -178,7 +179,15 @@ export class WorkAllocationService {
    * @param caseId The ID of the case to find tasks for.
    */
   public getTasksByCaseIdAndEventId(eventId: string, caseId: string, caseType: string, jurisdiction: string): Observable<TaskPayload> {
-    return this.http.get(`${this.appConfig.getWorkAllocationApiUrl()}/case/tasks/${caseId}/event/${eventId}/caseType/${caseType}/jurisdiction/${jurisdiction}`);
+    return this.http.get(`${this.appConfig.getWorkAllocationApiUrl()}/case/tasks/${caseId}/event/${eventId}/caseType/${caseType}/jurisdiction/${jurisdiction}`).pipe(catchError(error => {
+      if (error instanceof HttpErrorResponse) {
+        if (error.status === 401 || error.status === 403 || error.status === 404) {
```

Comment:
can check for error before error.status

Reply:
@connorpgpmcelroy Also if there is no code to be executed in the if statement should it be re-written, something like this?

```
const { status } = error;
if (status !== 401 && status !== 403 && status !== 404) {
  return throwError(error);
}
```